packages/main/cypress/specs/TimePickerClock.cy.tsx

```diff
@@ -0,0 +1,202 @@
+import TimePickerClock from "../../src/TimePickerClock.js";
+import Input from "../../src/Input.js";
+
+
+describe("Clock API", () => {
+
+	it("'disabled' property", () => {
+		cy.mount(
+			<>
+				<div style="display: inline-block; text-align: center; width: 18rem; height: 18rem; touch-action: none; margin: 2rem;">

```

Comment:
could be extracted into function which is going to be used into JSX to render the component